src/features/data/selectors/analytics.ts

```diff
@@ -16,11 +16,11 @@ export const selectUserDepositedTimelineByVaultId = createCachedSelector(
   (state: BeefyState, _vaultId: VaultEntity['id'], _address?: string) => state.user.analytics,
   (state: BeefyState, vaultId: VaultEntity['id'], _address?: string) => vaultId,
   (walletAddress, analyticsState, vaultId) =>
-    analyticsState.byAddress[walletAddress]?.timeline.byVaultId[vaultId] || []
+    analyticsState.byAddress[walletAddress.toLocaleLowerCase()]?.timeline.byVaultId[vaultId] || []
```

Comment:
```suggestion
    analyticsState.byAddress[walletAddress?.toLocaleLowerCase()]?.timeline.byVaultId[vaultId] || []
```

walletAddress can be null, any vault page will crash if no wallet is connected